### PR TITLE
Resolved IDENTITY-3951

### PR DESCRIPTION
**Pull request by IndunilRathnayake**

### CODE REVIEW

File: components/user-mgt/org.wso2.carbon.user.mgt.ui/src/main/java/org/wso2/carbon/user/mgt/ui/UserAdminClient.java [hunk 1]
```diff
@@ -230,10 +230,10 @@ public void setRoleUIPermission(String roleName, String[] rawResources) throws A
...
 
-    public void bulkImportUsers(String fileName, DataHandler handler, String defaultPassword)
+    public void bulkImportUsers(String userStoreDomain, String fileName, DataHandler handler, String defaultPassword)
             throws AxisFault {
         try {
-            stub.bulkImportUsers(fileName, handler, defaultPassword);
+            stub.bulkImportUsers(userStoreDomain, fileName, handler, defaultPassword);
         } catch (Exception e) {
             handleException(e);
         }
```
Comment (cdwijayarathna) [inline components/user-mgt/org.wso2.carbon.user.mgt.ui/src/main/java/org/wso2/carbon/user/mgt/ui/UserAdminClient.java]: Need to apply same changes to https://github.com/wso2/product-is/blob/master/modules/samples/identity-mgt/info-recovery-sample/src/main/java/org/wso2/sample/inforecovery/client/UserAdminClient.java

Comment (IndunilRathnayake) [inline components/user-mgt/org.wso2.carbon.user.mgt.ui/src/main/java/org/wso2/carbon/user/mgt/ui/UserAdminClient.java]: This change is applied.

File: components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/UserRealmProxy.java [hunk 2]
```diff
@@ -2092,14 +2086,20 @@ public void setRoleUIPermission(String roleName, String[] rawResources)
...
 
-    public void bulkImportUsers(String fileName, InputStream inStream, String defaultPassword)
+    public void bulkImportUsers(String userStoreDomain, String fileName, InputStream inStream, String defaultPassword)
             throws UserAdminException {
         try {
             BulkImportConfig config = new BulkImportConfig(inStream, fileName);
             if (defaultPassword != null && defaultPassword.trim().length() > 0) {
                 config.setDefaultPassword(defaultPassword.trim());
             }
+            if(StringUtils.isNotEmpty(userStoreDomain)){
```
Comment (lafernando) [inline components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/UserRealmProxy.java]: Fix code formatting.

Comment (IndunilRathnayake) [inline components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/UserRealmProxy.java]: Fixed in https://github.com/wso2/carbon-identity/pull/1601

**Pull request merged**